src/lua_console.cpp

```diff
@@ -0,0 +1,93 @@
+#include "lua_console.h"
+
+#include "catalua.h"
+#include "catacurse.h"
+
+#include "debug.h"
+#include "catacharset.h"
+
+lua_console::lua_console()
+{
+    cWin = newwin(lines, width, 0, 0);
```

Comment:
Nitpicking (there is not much difference in this case), these should be in the member initializer list